server/lists/adapters/case-summary.js

```diff
@@ -34,6 +34,22 @@ const createAdditionalFields = async (additionalFields = [], fetchList) => {
     return results;
 };
 
+const fetchChoices = async (choices, fetchList) => {
+    if (choices.includes(', ')) {
+        const choicesList = [];
+        const listNames = choices.split(', ');
+        for (var j = 0; j < listNames.length; j++) {
+            const fetchedList = await fetchList(listNames[j]);
+            for (var k = 0; k < fetchedList.length; k++) {
+                choicesList.push(fetchedList[k]);
+            }
```

Comment:
There's a few native functions in `Array.prototype` that might be useful to make more robust code that doesn't require a for-loop (and might be faster).

If you want to combine two arrays, the [`Array.prototype.concat()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) function might be better than a for loop. If you have an array of arrays, you can flatten it into a single array with `.flat()`.

Similarly, if you are wanting to perform an operation on every element in an array, [`Array.prototype.map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) might be a more readable choice.


(If you really wanted -- I wouldn't recommend this? -- you could collapse it all into a one-liner:)
```js
const choicesList = choices.split(', ').map((item) => await fetchList(item)).flat()
```